tests/h/cli/commands/authclient_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+
+from h import models
+from h.cli.commands import authclient as authclient_cli
+
+
+class TestAddCommand(object):
+
+    def test_it_creates_an_authclient(self, cli, cliconfig, db_session):
+        authclient = self._add_authclient(cli, cliconfig, db_session)
+        assert authclient
```

Comment:
It might be better to do: `assert authclient is not None` because that's what SQLAlchemy's `first()` returns if it can't find anything, otherwise any truthy value would pass this test.